cmd/subnetcmd/deploy.go

```diff
@@ -118,6 +118,10 @@ func deploySubnet(cmd *cobra.Command, args []string) error {
 	switch network {
 	case models.Local:
 		app.Log.Debug("Deploy local")
+		sc, err := app.LoadSidecar(chain)
+		if err != nil {
+			return fmt.Errorf("creation of chains and subnet was successful, but failed to load sidecar for update: %w", err)
```

Comment:
I think we need to change the error message since this is now before chain creation

Reply:
You are right of course. Updated.